kolibri/plugins/style_guide/assets/src/views/content/components/buttons.vue

```diff
@@ -200,8 +204,16 @@
 </style>
 `;
 
+  const api = require('!vue-doc!keen-ui/src/UiButton'); // eslint-disable-line
+
   module.exports = {
-    data: () => ({ codeExamplesTemplate })
+    components: {
+      'api-table': require('../api_table'),
+    },
+    data: () => ({ 
```

Comment:
no - for me the linter is complaining about some trailing space. E.g.:

![image](https://user-images.githubusercontent.com/2367265/27059800-4c4858ca-4f8d-11e7-9708-84ddcc1e0738.png)